test/Libraries/RevitIntegrationTests/MigrationTest.cs

```diff
@@ -68,7 +68,7 @@ public void TestMigration_Analyze_Measure()
         [Test, Category("Failure")]
         [TestModel(@".\empty.rfa")]
         public void TestMigration_Analyze_Render()
-        {
+        {   //There are some obsolete node in dyn script.
```

Comment:
The three tests of Migration tests have some obsolete node, but I don't know if this is what this test is expecting.